package.json

```diff
@@ -28,7 +28,8 @@
         "jsdom": "~3.1.0",
         "bluebird": "~2.9.1",
         "lodash": "~2.4",
-        "qs": "2.3.3"
+        "qs": "2.3.3",
```

Comment:
So what your're suggesting is that if we upgrade express, we should also upgrade these dependencies to match the version express has?